src/roadmap/release-checklist.md

```diff
@@ -73,15 +73,17 @@ version of our release for testing. Here is the current process to do so:
 
         cp -r workflow-dev workflow-$WORKFLOW_RELEASE_SHORT
         cp -r workflow-dev-e2e workflow-$WORKFLOW_RELEASE_SHORT-e2e
+        cp -r router-dev router-$WORKFLOW_RELEASE_SHORT
 
   4. Stage copies of all files needing release updates into the appropriate `workflow-$WORKFLOW_RELEASE_SHORT(-e2e)` chart directories:
 
         deisrel helm-stage --tag $WORKFLOW_RELEASE --stagingDir workflow-$WORKFLOW_RELEASE_SHORT workflow
         deisrel helm-stage --tag $WORKFLOW_RELEASE --stagingDir workflow-$WORKFLOW_RELEASE_SHORT-e2e e2e
+        deisrel helm-stage --tag $WORKFLOW_RELEASE --stagingDir router-$WORKFLOW_RELEASE_SHORT router
 
   5. Delete the `KUBERNETES_POD_TERMINATION_GRACE_PERIOD_SECONDS` env var from `workflow-$WORKFLOW_RELEASE_SHORT/tpl/deis-controller-rc.yaml`
 
-  6. Test the chart and make sure it installs:
+  6. Test the new Workflow chart and make sure it installs:
```

Comment:
fwiw, the change on this line is just meant to be specific about _which_ chart, since the steps above produce three.